aims/aim1.md

```diff
@@ -0,0 +1,44 @@
+# Aim 1:
+
+**Develop proof-of-concept deep learning methods to learn a latent space from single cell transcriptomic data.**
+
+## Background
+
+Deep unsupervised models can learn to approximate a data generating manifold often represented in a lower dimensional latent space.
```

Comment:
I am not sure. Based on how the aim 1 is structured in this PR I think it makes sense - but in an updated version of the aim, I don't think it does. My current thinking is:

aim 1 - train/troubleshoot the model
aim 3 - latent space arithmetic (which necessitates an eval to other latent space finding algorithms)